packages/clarity-js/src/layout/mutation.ts

```diff
@@ -29,14 +36,19 @@ export function start(): void {
     // using javascript API is that it doesn't trigger DOM mutation and therefore we
     // need to override the insertRule API and listen for changes manually.
     CSSStyleSheet.prototype.insertRule = function(rule: string, index?: number): number {
-      let value = insertRule.call(this, rule, index);
-      generate(this.ownerNode, Constant.CHARACTER_DATA);
-      return value;
+      try {
+        let value = insertRule.call(this, rule, index);
+        schedule(this.ownerNode);
+        return value;
+      } catch (error) {
+        log.log(Code.CssRules, error, Severity.Info);
+        return null;
```

Comment:
Swallowing this error is causing issues per the details documented in this PR https://github.com/microsoft/clarity/pull/62 

Updating your beta branch that is shipped to customers to not swallow this error would be highly appreciated.